xwiki-commons-core/xwiki-commons-configuration/xwiki-commons-configuration-default/src/main/resources/META-INF/components.txt

```diff
@@ -0,0 +1,3 @@
+org.xwiki.configuration.internal.SystemPropertiesConfigurationSource
+org.xwiki.configuration.internal.SystemEnvConfigurationSource
+org.xwiki.configuration.internal.SystemConfigurationSource
```

Comment:
Hmmm it's Github diff now showing the last carriage return? AFAIR we have a check for assessing there's a final empty line

Reply:
It's strange, I do have a new line at the end locally, but I don't see it on https://github.com/xwiki/xwiki-commons/blob/feature-systemconfiguration/xwiki-commons-core/xwiki-commons-configuration/xwiki-commons-configuration-default/src/main/resources/META-INF/components.txt.